app/javascript/packs/flagUserModal.js

```diff
@@ -0,0 +1,158 @@
+import { request } from '../utilities/http';
+
+const modalSnackbarHTML = `
```

Comment:
As discussed this afternoon, is this code being used at all, or do the existing alerts you have  cover the "ghetto" snackbar items for now. If so, feel free to remove this code.

Reply:
This is still currently being used, so we'll keep it for now. Going to try to refactor and use the snackbar component but it's more of a stretch goal at this point.